lib/measures/tbd/resources/psi.rb

```diff
@@ -147,16 +147,33 @@ def append(p)
       end
     end
   end
+
+  def complete?(s)
```

Comment:
Normally a `?` method would return true or false rather than raise an exception

Reply:
Noted. I simplify this accordingly, but maybe wait when I'll log mild/severe warnings on file.